cputree.c

```diff
@@ -299,6 +299,7 @@ static void do_one_cpu(char *path)
 		return;
 	}
 
+	cpus_clear(package_mask);
```

Comment:
No, this is completely unnecessecary.  package_mask is cleared as part of cpumask_parse_user.  In commit c7122666e0f72634512284fe90258daf1dca5914 you should have just removed the call to cpu_set entirely when you converted to the use of the process_one_line function.  Please just remove the call to cpu_set

Reply:
The comment "try to read the package mask; if it doesn't exist assume solitary", so it means process_one_line() maybe will fail, it need a default package_mask. is that true? thanks.

Reply:
Yes, if process_one_line fails, you will then need to clear and set the mask in the error condition.  either way though, you don't need to clear it before hand.

Reply:
ok, I have update the patch, thanks.